_posts/en/newsletters/2023-05-17-newsletter.md

```diff
@@ -0,0 +1,217 @@
+      unable to provide a more detailed description at this time, but
+      anyone interested in the topic may want to read the summary.
+
+    - [Project meta discussion][] summarizes a varied discussion about
+      the projects goals and how to achieve them despite many
+      challenges, both internal and external.  Some of the discussion
+      has already led to experimental changes in the project's
+      management, such as a more project-focused approach for the next
+      major release after version 25.
+
+## Waiting for confirmation #1: why do we have a mempool?
+
+_The first segment in a limited weekly series about transaction relay,
+mempool inclusion, and mining transaction selection---including why
+Bitcoin Core has a more restrictive policy than allowed by consensus and
+how wallets can use that policy most effectively._
+
+{% include specials/policy/en/01-why-mempool.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- [Core Lightning 23.05rc2][] is a release candidate for the next
+  version of this LN implementation.
+
+- [Bitcoin Core 24.1rc2][] is a release candidate for a maintenance
+  release of the current version of Bitcoin Core.
+
+- [Bitcoin Core 25.0rc1][] is a release candidate for the next major
+  version of Bitcoin Core.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #26076][] RPC methods that show derivation paths for
+  public keys now use `h` instead of a single-quote `'` to indicate a
+  hardened derivation step. Note that this changes the descriptor
+  checksum. When handling descriptors with private keys, the same symbol
+  is used as when the descriptor was generated or imported. For legacy
+  wallets the `hdkeypath` field in `getaddressinfo` and the
+  serialization format of wallet dumps remain unchanged.
+
+- [Bitcoin Core #27608][] p2p: Avoid prematurely clearing download state for other peers FIXME:harding
+
+- [LDK #2286][] Create and Sign PSBTs for spendable outputs FIXME:harding
+
+- [LDK #1794][] dunxen/2022-10-dualfunding-act-1 FIXME:harding
+
+- [Libsecp256k1 #1066][] Abstract out and merge all the magnitude/normalized logic FIXME:harding
+
+- [Libsecp256k1 #1299][] Infinity handling: ecmult_const(infinity) works, and group verification FIXME:harding
```

Comment:
@sipa thanks!  I think while you were writing your summary above, I pushed a commit which dropped our mention of both PRs.  I'm sorry to have wasted your time.  For background, I usually do a quick skim of all commits added to our monitored projects each week to make a list of anything that might be interesting.  Then I research each of the items on the list until I decide that it's either not interesting or I think I understand it well enough to describe it (or ask one of our other contributors to describe it).

In most weeks, nobody besides me ever sees my list of might-be-interesting PRs; this week was an exception because I was running behind schedule.  Again, thanks for the summary!